app/views/layouts/application.html.erb

```diff
@@ -8,7 +8,7 @@
     <meta property="og:title" content="<%= meta_title %>">
     <meta property="og:type" content="website">
     <meta property="og:url" content="<%= request.original_url %>">
-    <meta property="og:image" content="<%= meta_image %>">
+    <meta property="og:image" content="https://ibb.co/SBMCB9g">
```

Comment:
Can we store this locally (within assets) can point to the local version instead?

Reply:
Thats what we had before, but it seems in the testing on kitt they want it stored as an absolute URL